### PR TITLE
Separate Inexor Flex in it's own repository

### PR DESCRIPTION
**Pull request by aschaeffer**
Solves  #360 

- [x] Created a new repository: [Inexor Flex](/inexor-game/flex)
- [x] Added Inexor Flex as a submodule of Inexor Core
- [x] Inexor Flex is able to start Inexor Core
- [x] Updated start scripts

### CODE REVIEW

**Comment by a-teammate** [comment]
i think you forgot to remove the node folder in the commit

**Comment by aschaeffer** [comment]
Also the startup scripts has to be updated

File: tool/install_flex_dependencies.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+cd  flex
+npm install
+npm install @inexor-game/connector
```
Comment (Fohlen) [inline tool/install_flex_dependencies.sh]: Why would `npm install` not install all necessary dependencies at the first time?

Comment (aschaeffer) [inline tool/install_flex_dependencies.sh]: If this work, then just change it.

Comment (Fohlen) [inline tool/install_flex_dependencies.sh]: This'll only be problematic if you change the modules after install time which is usually not the case for end-users.

**Pull request merged**